### PR TITLE
lxd: add core scheduling support

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: lxd/instance/drivers/driver_lxc.go [hunk 1]
```diff
@@ -739,6 +739,13 @@ func (d *lxc) initLXC(config bool) error {
 		}
 	}
 
+	if d.state.OS.CoreScheduling {
+		err = lxcSetConfigItem(cc, "lxc.sched.core", "1")
```
Comment (brauner) [inline lxd/instance/drivers/driver_lxc.go:743]: In the current implementation we always make use of core scheduling. Do we want to be able to configure this through a config option?

Comment (brauner) [reply lxd/instance/drivers/driver_lxc.go:743]: *lxd config option

Comment (stgraber) [reply lxd/instance/drivers/driver_lxc.go:743]: For now, just always doing it seems fine to me I think.

Comment (brauner) [reply lxd/instance/drivers/driver_lxc.go:743]: sounds good

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**